src/expressions/visitors/CompareVisitor.cpp

```diff
@@ -22,15 +22,24 @@
 #include <antares/expressions/visitors/CompareVisitor.h>
 
 template<class T, class V>
-static bool compareBinaryNode(V& visitor,
+static bool compareParentNode(V& visitor,
                               const T* node,
                               const Antares::Expressions::Nodes::Node* other)
 {
-    if (const T* other_node = dynamic_cast<const T*>(other))
+    if (const auto* other_node = dynamic_cast<const T*>(other))
```

Comment:
Why not `T*` ? It's more clear & shorter than `auto*`

Reply:
alright!